vscode-antimony/src/server/main.py

```diff
@@ -125,6 +128,7 @@ def sbml_str_to_ant_str(ls: LanguageServer, args):
     antimony.freeAll()
     ant = antimony.loadSBMLString(sbml)
     if ant < 0:
+        vscode_logger.info("SBML conversion failed")

```

Comment:
Delete this logger. If you want to display this to user, use vscode error message